--- a/configure.ac
+++ b/configure.ac
@@ -307,7 +307,12 @@
     [], [with_daos=no])
 AS_IF([test "x$with_daos" != xno], [
     DAOS="yes"
-    LDFLAGS="$LDFLAGS -L$with_daos/lib64 -Wl,--enable-new-dtags -Wl,-rpath=$with_daos/lib64"
+    if test -d $with_daos/lib/x86_64-linux-gnu/; then
+        LIB_DIR=$with_daos/lib/x86_64-linux-gnu
+    else
+        LIB_DIR=$with_daos/lib64
+    fi
+    LDFLAGS="$LDFLAGS -L$LIB_DIR -Wl,--enable-new-dtags -Wl,-rpath=$LIB_DIR -lmercury"
     CPPFLAGS="$CPPFLAGS -I$with_daos/include"
     AC_CHECK_HEADERS(gurt/common.h,, [unset DAOS])
     AC_CHECK_HEADERS(daos.h,, [unset DAOS])
